Source/TestDafny/MultiBackendTest.cs

```diff
@@ -22,6 +22,9 @@ public class ForEachCompilerOptions {
   [Value(1, MetaName = "Dafny CLI arguments", HelpText = "Any arguments following '--' will be passed to the dafny CLI unaltered.")]
   public IEnumerable<string> OtherArgs { get; set; } = Array.Empty<string>();
 
+  [Option("run-fails", HelpText = "Whether the running program should return a non-zero exit code")]
+  public bool RunShouldFail { get; set; } = false;
```

Comment:
Good feature to add, but would strongly prefer `--expect-exit-code` instead, since we already have that option on `for-each-resolver`

Reply:
I can't pass in an exit code because I don't know what it will be. Dafny does not define what the exit code will be for programs that do not pass runtime contract checks. It could be different per target. All I know it that it won't be a success exit code.

Reply:
Ah fair! I can live with this then.